app/Http/Controllers/Api/WaitingRoomsController.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Gladiator\Http\Controllers\Api;
+
+use Illuminate\Http\Request;
+use Gladiator\Http\Controllers\Controller;
+use Gladiator\Models\WaitingRoom;
+
+class WaitingRoomsController extends Controller
+{
+
+    /**
+     * Get a waiting room for the specified campaign ID
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function getWaitingRoom(Request $request)
+    {
+        $this->validate($request, [
+            'campaign_id' => 'required|integer'
+        ]);
+
+        $campaign_id = $request->input('campaign_id');
+        $waiting_room = WaitingRoom::find($campaign_id);
```

Comment:
Oh I know its not required, im just used to doing this in Drupal since that seemed to be the convention there